src/components/Tabs/Tabs.jsx

```diff
@@ -1 +1,31 @@
-export const Tabs = () => {};
+import cn from 'classnames';
+
+export const Tabs = ({ tabs, activeTabId, onTabSelected: chosen }) => {
```

Comment:
```suggestion
export const Tabs = ({ tabs, activeTabId, onTabSelected }) => {
```
`chosen` doesn't describe function, `onTabSelected` looks much better